docs/api/index.md

```diff
@@ -32,8 +32,7 @@ The API exists to allow other software systems to **read** and **write** Sara Al
 The examples above are in terms of Monitorees, but the API also supports Symptom Reports and Lab Results. However, these entities are currently only supported for reading data.
 
 ### For Monitorees
-**A client can read and write the following data elements:**<br>Workflow, First Name, Middle Name, Last Name, Date of Birth, Sex, White, Black or African American, American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Ethnicity, Primary Language, Interpretation Requirement, Address 1, Town/City, State, Zip, Country, Preferred Reporting Method, Preferred Contact Time, Primary Telephone Number, Secondary Telephone Number, E-mail Address, Last Date of Exposure, Symptom Onset Date, Monitoring Status, Assigned Jurisdiction, Monitoring Plan, Assigned User, Additional Planned Travel Start Date, Additional Planned Travel Notes, Port of Origin, Date of Departure, Flight or Vessel Number, Flight or Vessel Carrier, Date of Arrival, Travel Related Notes, Exposure Notes, Primary Telephone Type, Secondary Telephone Type, State/Local ID
-
+**A client can read and write the following data elements:**<br>Workflow, First Name, Middle Name, Last Name, Date of Birth, Sex, White, Black or African American, American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Ethnicity, Primary Language, Interpretation Requirement, Address 1, Address 2, Town/City, State, Zip, Country, Address 1 (Foreign), Address 2 (Foreign), Address 3 (Foreign), Town/City (Foreign), State/Province (Foreign), Postal Code (Foreign), Country (Foreign), Preferred Reporting Method, Preferred Contact Time, Primary Telephone Number, Secondary Telephone Number, E-mail Address, Last Date of Exposure, Symptom Onset Date, Monitoring Status, Assigned Jurisdiction, Monitoring Plan, Assigned User, Additional Planned Travel Start Date, Additional Planned Travel Notes, Port of Origin, Date of Departure, Flight or Vessel Number, Flight or Vessel Carrier, Date of Arrival, Travel Related Notes, Exposure Notes, Primary Telephone Type, Secondary Telephone Type, State/Local ID
```

Comment:
Can you please also add a section on the release notes page for 1.22 and include this change?

Reply:
Yes, addressed in https://github.com/SaraAlert/SaraAlert/pull/686/commits/8d2d37f6647f455e9f4d4a2a3aef6b5324a6e634. I also added a bullet in the Release Notes about the update side effects change, and updated the 1.23 coming soon.

Reply:
This isn't going to be in 1.22 but rather 1.23 - so those release notes would need to be updated. This branch should also now target 1.23.0 instead.

Reply:
Ok I've updated the target, and removed that documentation commit. I will be making a separate documentation PR to update the release notes for 1.22.0.